flow/core/kernel/scenario/base.py

```diff
@@ -57,6 +61,12 @@ def __init__(self, master_kernel):
         self.network = None
         self.orig_name = None
         self.name = None
+        self.edgestarts = None
+        self.internal_edgestarts = None
+        self.intersection_edgestarts = None
+        self.internal_edgestarts_dict = None
```

Comment:
Are we keeping the internal edge_starts object? What function does it serve?

Reply:
ya.. I have an idea of how to fix it, but it'll take some time. Will prioritize ASAP